core/common/src/test/java/alluxio/AlluxioTestDirectory.java

```diff
@@ -90,12 +81,19 @@ private static void cleanUpOldFiles(File dir) {
     File[] files = dir.listFiles();
     for (File file : files) {
       if (!FileUtils.isFileNewer(file, cutoffTimestamp)) {
-        try {
-          alluxio.util.io.FileUtils.deletePathRecursively(file.getAbsolutePath());
-        } catch (Exception e) {
-          LOG.warn("Failed to delete {} : {}", file.getAbsolutePath(), e.getMessage());
-        }
+        delete(file);
       }
     }
   }
+
+  private static void delete(File file) {
+    if (!file.exists()) {
+      return;
+    }
+    try {
+      alluxio.util.io.FileUtils.deletePathRecursively(file.getAbsolutePath());
```

Comment:
Can we import this?

Reply:
we use apache common `FileUtils` elsewhere in the file. I've switched to using `FileUtils.deleteDirectory`